lib/portfinder.js

```diff
@@ -155,7 +155,7 @@ exports.getSocket = function (options, callback) {
     options = {};
   }
 
-  options.mod  = options.mod    || 0755;
```

Comment:
@dcharbonnier - thanks for the PR

That said, I *am* open to changing this, but given I had to hunt down the reasoning, I'm a little weary. The node source code says:

> // Maintainers, keep in mind that octal literals are not allowed
// in strict mode. Use the decimal value and add a comment with
// the octal value. Example:
//
//   var mode = 438; /* mode=0666 */

which makes sense as to why when I ran 

`parseInt(755, 8)` i get 493

So yes, ur change is correct. However, we can also use '0755' and node will take care of that for us, avoiding the confusion and preventing the error in strict mode. 

I suppose last note worth mentioning is that 'use strict' is not used within the source, so perhaps that's step 1. I'm willing to pull this in though if we simply use '0755' ATM, although we should make sure this behavior has not been removed in new versions of node (although that sounds very unlikely)

(ps - here is where they make that conversion for us in an old 0.x archive (found this on stack overflow)): https://github.com/nodejs/node-v0.x-archive/blob/c7f424e44b7e61b89f91cd344599c3d1cdfb88fe/lib/fs.js#L422-L423

Thanks!